projects/ui/src/lib/components/po-field/po-lookup/po-lookup.component.spec.ts

```diff
@@ -324,6 +324,34 @@ describe('PoLookupComponent:', () => {
       expect(component.inputEl.nativeElement.value).toBe('123 - teste');
     });
 
+    it('setInputValueWipoieldFormat: should define the input value and the old value with the value returned from formatFields with the fieldFormat being an array', () => {
```

Comment:
```suggestion
    it('setInputValueWipoieldFormat: should set input value and old value with formatedField  if fieldFormat is array', () => {
```

obs: Quando passar de 120 linhas devemos quebra-la.